source/includes/options-mongorestore.yaml

```diff
@@ -239,6 +240,12 @@ description: |
   level of the dump directory for :program:`mongorestore --oplogReplay
   <mongorestore>` to replay.
 
+  :program:`mongorestore --oplogReplay <mongorestore>` must restore a
+  full dump of a :term:`replica set` member created using 
+  :program:`mongodump --oplog <mongodump>`. You cannot use 
+  :option:`--database` or :option:`--collection` with {{role}} to 
```

Comment:
Are these the only two options? There are a number of include/exclude options related to namespaces as well - does --oplogReplay work with those?

If so, rather than having a full list I think you could make this generic:

"``--oplogReplay`` restores the entire dump, and is incompatible with limiter options such as --database, --collection, or --nsExclude"